src/components/implementation/no_interface/cFE_booter/cfe_psp_eeprom.c

```diff
@@ -0,0 +1,352 @@
+/*
```

Comment:
Did you have to make changes in this file? If not, perhaps just a soft-link from the extern directory would be better.